src/Breadcrumbs.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bulma;
+
+use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Html\Html;
+use Yiisoft\Widget\Exception\InvalidConfigException;
+
+/**
+ * The Bulma breadcrumb is a simple navigation component that only requires
+ *
+ * For example,
+ *
+ * ```php
+ * echo Breadcrumbs::widget()
+ *     ->links(['label' => !empty($this->params['breadcrumbs']) ? $this->params['breadcrumbs'] : []]);
+ * ```
+ */
+class Breadcrumbs extends Widget
+{
+    private bool $encodeLabels = true;
+    private array $homeLink = [];
+    private string $itemTemplate = "<li>{link}</li>\n";
```

Comment:
You might want to use a constant 

```php
private string $itemTemplate = "<li>{link}</li>" . PHP_EOL;
```

Reply:
If I use a constant you will not be able to configure the item template, the idea is that it be configurable, you can change the order or the tag.